src/alert-converter/alert-converter.service.ts

```diff
@@ -1,20 +1,32 @@
+import { ConfigService } from '@nestjs/config';
 import { LogMessageFormat, LogType } from 'logging-format';
 /**
- * Class for Converting an Alert into a LogMessage. 
- * TODO: Make things more generic
+ * Class for Converting an Alert into a LogMessage.  * 
  */
 export class AlertConverterService {
+    prometheusUrl: string;
+    windowsExporterUrl: string;
+    constructor(private configService: ConfigService) {
+    }
 
     /**
-     * Takes Alert in correct JSON Format and converts into LogMessage Array.
+     * Takes Alert in correct JSON Format and converts into LogMessage Array. 
+     * If the LogType should be included, the description of a rule in rules.yml needs to contain the LogType.
```

Comment:
blank line zwischen short und long description :) 
```suggestion
     * Takes Alert in correct JSON Format and converts into LogMessage Array. 
     * 
     * If the LogType should be included, the description of a rule in rules.yml needs to contain the LogType.
```